packages/types/src/deneb/types.ts

```diff
@@ -47,9 +47,9 @@ export type LightClientStore = ValueOf<typeof ssz.LightClientStore>;
 
 export type ProducedBlobSidecars = Omit<BlobSidecars, "signedBlockHeader" | "kzgCommitmentInclusionProof">;
 
-export type BlockContents = ValueOf<typeof ssz.BlockContents>;
-export type SignedBlockContents = ValueOf<typeof ssz.SignedBlockContents>;
```

Comment:
these types are defined in the spec [here](https://github.com/ethereum/beacon-APIs/blob/a70801d0b4714d76e2b8e0d9dccc3f8bada84275/types/deneb/block_contents.yaml#L16-L38)